.env.local

```
@@ -0,0 +1,2 @@
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_YWxlcnQtY2FyZGluYWwtOTYuY2xlcmsuYWNjb3VudHMuZGV2JA
+CLERK_SECRET_KEY=sk_test_8SC4KFhIIehVKd5LdttDs5Tw0VeSQakYBr3tNecLSb
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)